circleci: failed tests: TestManualReplication

The following test appears to have failed:

[#22891](https://circleci.com/gh/cockroachdb/cockroach/22891):

```
I160916 20:14:59.543393 96 kv/txn_coord_sender.go:251  [n2] txn coordinator: 0.00 txn/sec, 100.00/0.00/0.00/0.00 %cmmt/cmmt1pc/abrt/abnd, 0s/0s/0s avg/σ/max duration, 0.0/0.0/0 avg/σ/max restarts (0 samples)
W160916 20:14:59.908819 147 gossip/infostore.go:295  [n2] node unavailable; try another peer
I160916 20:14:59.940522 161 kv/txn_coord_sender.go:251  [n3] txn coordinator: 0.00 txn/sec, 100.00/33.33/0.00/0.00 %cmmt/cmmt1pc/abrt/abnd, 0s/0s/0s avg/σ/max duration, 0.0/0.0/0 avg/σ/max restarts (0 samples)
W160916 20:14:59.974658 255 gossip/infostore.go:295  [n3] node unavailable; try another peer
I160916 20:15:02.119900 59 storage/replica.go:2294  [n1,s1,r2{/Table/51-/Max}] pending commands: refurbished 0, reproposing 1 (at 17.6); reasonTicks
panic: test timed out after 2m0s

goroutine 466 [running]:
panic(0x1415600, 0xc420cc0ee0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
testing.startAlarm.func1()
	/usr/local/go/src/testing/testing.go:918 +0x10b
created by time.goFunc
	/usr/local/go/src/time/sleep.go:154 +0x44

goroutine 1 [chan receive, 2 minutes]:
testing.(*T).Run(0xc42016a0c0, 0x15b14ec, 0x15, 0x16fd3d0, 0xc420277ce0)
	/usr/local/go/src/testing/testing.go:647 +0x316
testing.RunTests.func1(0xc42016a0c0)
	/usr/local/go/src/testing/testing.go:793 +0x6d
testing.tRunner(0xc42016a0c0, 0xc42043ae00)
	/usr/local/go/src/testing/testing.go:610 +0x81
testing.RunTests(0x16fe460, 0x1ec84e0, 0x4, 0x4, 0x224fb28)
	/usr/local/go/src/testing/testing.go:799 +0x2f5
testing.(*M).Run(0xc42043aef8, 0x224f700)
	/usr/local/go/src/testing/testing.go:743 +0x85
github.com/cockroachdb/cockroach/testutils/testcluster.TestMain(0xc42043aef8)
	/go/src/github.com/cockroachdb/cockroach/testutils/testcluster/main_test.go:34 +0x92
main.main()
	github.com/cockroachdb/cockroach/testutils/testcluster/_test/_testmain.go:58 +0xc6

goroutine 17 [syscall, 2 minutes, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2086 +0x1

goroutine 20 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x2216ca0)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1032 +0x77
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:613 +0x95

goroutine 23 [semacquire, 1 minutes]:
sync.runtime_Semacquire(0xc42120550c)
	/usr/local/go/src/runtime/sema.go:47 +0x30
sync.(*WaitGroup).Wait(0xc421205500)
	/usr/local/go/src/sync/waitgroup.go:131 +0x97
github.com/cockroachdb/cockroach/testutils/testcluster.(*TestCluster).stopServers(0xc42013a060)
	/go/src/github.com/cockroachdb/cockroach/testutils/testcluster/testcluster.go:98 +0x12a
github.com/cockroachdb/cockroach/testutils/testcluster.(*TestCluster).(github.com/cockroachdb/cockroach/testutils/testcluster.stopServers)-fm()
	/go/src/github.com/cockroachdb/cockroach/testutils/testcluster/testcluster.go:172 +0x2a
github.com/cockroachdb/cockroach/util/stop.CloserFn.Close(0xc420add850)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:87 +0x19
--
testing.tRunner(0xc42016a180, 0x16fd3d0)
	/usr/local/go/src/testing/testing.go:610 +0x81
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:646 +0x2ec

goroutine 7 [chan receive, 2 minutes]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc42013a5a0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:390 +0x44
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:391 +0x6eb

goroutine 9 [select, 2 minutes]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:340 +0x3da
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420258320)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 10 [select]:
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).startStats(0xc42001ad10)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:208 +0xa63
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).(github.com/cockroachdb/cockroach/kv.startStats)-fm()
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:193 +0x2a
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4202526c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 11 [select]:
github.com/cockroachdb/cockroach/storage.NewRaftTransport.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:182 +0xe03
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420252a10)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 12 [select, 1 minutes]:
github.com/cockroachdb/cockroach/sql.(*LeaseManager).RefreshLeases.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/lease.go:1143 +0xcc1
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201d9ec0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 13 [select, 1 minutes]:
github.com/cockroachdb/cockroach/sql.NewExecutor.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/executor.go:274 +0x175
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc42054a4e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 14 [chan receive, 2 minutes]:
github.com/cockroachdb/cockroach/util/netutil.MakeServer.func2()
	/go/src/github.com/cockroachdb/cockroach/util/netutil/net.go:100 +0x70
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420258780)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 15 [chan receive, 2 minutes]:
github.com/cockroachdb/cockroach/util/netutil.MakeServer.func2()
	/go/src/github.com/cockroachdb/cockroach/util/netutil/net.go:100 +0x70
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4202587e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 37 [chan receive, 1 minutes]:
github.com/cockroachdb/cockroach/server.(*Server).Start.func6()
	/go/src/github.com/cockroachdb/cockroach/server/server.go:408 +0xcc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420258a20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 42 [select, 1 minutes]:
github.com/cockroachdb/cockroach/gossip.(*Gossip).manage.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:948 +0x2fb
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420253340)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 43 [select, 2 minutes]:
github.com/cockroachdb/cockroach/storage.(*bookie).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/reservation.go:274 +0x217
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420258ce0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 62 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).raftSnapshotStatusLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:2736 +0x211
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420013910)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 55 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*bookie).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/reservation.go:274 +0x217
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420692520)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 60 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc420682fc0, 0xc420176090)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cc5c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 61 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).raftTickLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:2706 +0x334
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420013900)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 59 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc420682fc0, 0xc420176090)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cc5a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 57 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc420682fc0, 0xc420176090)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cc560)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 81 [select]:
github.com/cockroachdb/cockroach/server.(*Node).startComputePeriodicMetrics.func1()
	/go/src/github.com/cockroachdb/cockroach/server/node.go:643 +0x252
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd6e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 58 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc420682fc0, 0xc420176090)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cc580)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 64 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:975 +0x165
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cc660)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 65 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1050 +0x314
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420013970)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 66 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1069 +0x314
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420013980)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 79 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd620)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 82 [select]:
github.com/cockroachdb/cockroach/server.(*Node).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/server/node.go:603 +0x4cf
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201dab40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 73 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd520)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 74 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd540)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 75 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd560)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 76 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd580)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 77 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd5a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 80 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc420696180, 0xecf6e4950, 0xc41eff8580, 0x2215c00, 0xc420185640, 0xc420176090, 0xc4201c4240, 0xc42062e6c8)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:201 +0x237
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc4201c4240, 0xc4201494a0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:259 +0x7c
github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc42011aa80, 0xc42020a040)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:251 +0x21b
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:261 +0x233
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd640)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 84 [select]:
github.com/cockroachdb/cockroach/server.(*Server).startSampleEnvironment.func1()
	/go/src/github.com/cockroachdb/cockroach/server/server.go:624 +0x16b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd7c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 85 [select]:
github.com/cockroachdb/cockroach/ts.(*poller).start.func1()
	/go/src/github.com/cockroachdb/cockroach/ts/db.go:90 +0x16f
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc420225370)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 86 [select]:
github.com/cockroachdb/cockroach/server.(*Node).startWriteSummaries.func1()
	/go/src/github.com/cockroachdb/cockroach/server/node.go:679 +0x309
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd7e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 87 [select, 1 minutes]:
github.com/cockroachdb/cockroach/sql.(*SchemaChangeManager).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/schema_changer.go:704 +0x12c1
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201cd800)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 91 [select]:
github.com/cockroachdb/cockroach/rpc.(*Context).runHeartbeat(0xc420134900, 0xc42000bd00, 0xc420253290, 0xf, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:294 +0x645
github.com/cockroachdb/cockroach/rpc.(*Context).GRPCDial.func1.1.1()
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:217 +0x69
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc420176090, 0xc4201d8270)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 92 [chan receive, 1 minutes]:
database/sql.(*DB).connectionOpener(0xc42016d4a0)
	/usr/local/go/src/database/sql/sql.go:730 +0x4a
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x1e9

goroutine 93 [chan receive, 1 minutes]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc42000dc20)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:390 +0x44
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:391 +0x6eb

goroutine 95 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:340 +0x3da
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc4204998c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 96 [select]:
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).startStats(0xc420211d90)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:208 +0xa63
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).(github.com/cockroachdb/cockroach/kv.startStats)-fm()
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:193 +0x2a
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc42015cbd0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 97 [select]:
github.com/cockroachdb/cockroach/storage.NewRaftTransport.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/raft_transport.go:182 +0xe03
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc42015cc10)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 98 [select, 1 minutes]:
github.com/cockroachdb/cockroach/sql.(*LeaseManager).RefreshLeases.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/lease.go:1143 +0xcc1
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc42014ec90)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 99 [select, 1 minutes]:
github.com/cockroachdb/cockroach/sql.NewExecutor.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/executor.go:274 +0x175
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc42013a960)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 100 [chan receive, 1 minutes]:
github.com/cockroachdb/cockroach/util/netutil.MakeServer.func2()
	/go/src/github.com/cockroachdb/cockroach/util/netutil/net.go:100 +0x70
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc420499cc0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 101 [chan receive, 1 minutes]:
github.com/cockroachdb/cockroach/util/netutil.MakeServer.func2()
	/go/src/github.com/cockroachdb/cockroach/util/netutil/net.go:100 +0x70
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc420499d20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 106 [chan receive, 1 minutes]:
github.com/cockroachdb/cockroach/server.(*Server).Start.func6()
	/go/src/github.com/cockroachdb/cockroach/server/server.go:408 +0xcc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc420499f60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 111 [select]:
github.com/cockroachdb/cockroach/gossip.(*Gossip).manage.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:948 +0x2fb
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc42015da50)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 113 [select, 1 minutes]:
github.com/cockroachdb/cockroach/storage.(*bookie).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/reservation.go:274 +0x217
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc42042a900)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 118 [select]:
github.com/cockroachdb/cockroach/rpc.(*Context).runHeartbeat(0xc42064ca80, 0xc42045e200, 0xc420225b80, 0xf, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:294 +0x645
github.com/cockroachdb/cockroach/rpc.(*Context).GRPCDial.func1.1.1()
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:217 +0x69
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc420484300)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 150 [select]:
github.com/cockroachdb/cockroach/ts.(*poller).start.func1()
	/go/src/github.com/cockroachdb/cockroach/ts/db.go:90 +0x16f
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc4208c6f80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 149 [select]:
github.com/cockroachdb/cockroach/server.(*Server).startSampleEnvironment.func1()
	/go/src/github.com/cockroachdb/cockroach/server/server.go:624 +0x16b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc420ae7120)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 147 [select]:
github.com/cockroachdb/cockroach/server.(*Node).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/server/node.go:603 +0x4cf
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc4208a44b0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 146 [select]:
github.com/cockroachdb/cockroach/server.(*Node).startComputePeriodicMetrics.func1()
	/go/src/github.com/cockroachdb/cockroach/server/node.go:643 +0x252
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4206498c0, 0xc420ae7080)
	/go/src/github.com
```
Please assign, take a look and update the issue accordingly.

--- a/storage/replica.go
+++ b/storage/replica.go
@@ -1564,6 +1564,14 @@ func (r *Replica) addWriteCmd(
 				} else {
 					log.Warningf(ctx, "unable to cancel expired Raft command %s", ba)
 				}
+			case <-r.store.stopper.ShouldQuiesce():
+				// If we can't abandon this command, the surrounding loop will
+				// run hot on this path, but not being able to abandon implies
+				// that the request is being processed and should be available
+				// momentarily.
+				if tryAbandon() {
+					pErr = roachpb.NewError(&roachpb.NodeUnavailableError{})
+				}
 			}
 		}
 	} else {

